src/games/strategy/triplea/delegate/AirBattle.java

```diff
@@ -761,14 +762,8 @@ private void notifyCasualties(final GUID battleID, final IDelegateBridge bridge,
       public void run() {
         try {
           getRemote(firingPlayer, bridge).confirmEnemyCasualties(battleID, "Press space to continue", hitPlayer);
-        } catch (final ConnectionLostException cle) {
-          // somone else will deal with this
-          // System.out.println(cle.getMessage());
-          // cle.printStackTrace(System.out);
-        } catch (final GameOverException e) {
-          // ignore
         } catch (final Exception e) {
-          // ignore
+          ClientLogger.logQuietly(e);
```

Comment:
I compared this `//ignore` against the other locations, this is the only one where it seems pretty clear we should be logging something. Hence why the others are not modified.